### PR TITLE
PR review required

### PR DESCRIPTION
**Pull request by thesudeshdas**
please review my project1 code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# project1
\ No newline at end of file
+# project1
+
+making some changes
```
Comment (thesudeshdas) [inline README.md:3]: Your code is shit

**Review by thesudeshdas — COMMENTED** [review]
Review done

File: README.md (hunk 1 above)
Comment (ardourApeX) [inline README.md:3]: You can some more lines of code

**Review by ardourApeX — COMMENTED** [review]
Rest of the work is fine

**Pull request merged**